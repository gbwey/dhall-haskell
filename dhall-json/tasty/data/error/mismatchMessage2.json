[ 1, 2, 3, "oh oh" ]
