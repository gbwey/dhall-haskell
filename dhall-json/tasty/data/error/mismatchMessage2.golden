[1;31m
Error: [0m$[3]: Dhall type expression and JSON value do not match:

Expected Dhall type:
Natural

JSON:
"oh oh"
